packages/augur-ui/src/modules/common/form.styles.less

```diff
@@ -1168,6 +1167,7 @@ div.FormDropdownActive {
 
     .SingleDatePicker_picker {
       margin-top: -1px;
```

Comment:
Also, recently John pointed out negative margin values are a code smell. Can you investigate if this negative is needed

Reply:
its needed so their isn't space below the arrow